src/main/java/io/lettuce/core/RedisCommandBuilder.java

```diff
@@ -2107,6 +2107,14 @@ Command<K, V, Map<K, Long>> pubsubNumsub(K... pattern) {
         return createCommand(PUBSUB, (MapOutput) new MapOutput<K, Long>((RedisCodec) codec), args);
     }
 
+    Command<K, V, Map<K, Long>> pubsubShardNumsub(K... pattern) {
```

Comment:
Is there's the reason why in this signature argument called `pattern` and not `channels` as in other places?

Reply:
Totally, let me address that too.